### ISSUE TITLE
[Enhancement] Customizable options menus?

### ISSUE THREAD

**Issue by iKaneki-Ken** [lumien231/Custom-Main-Menu#82]
Would it be possible to make the Options, Mods, and Singleplayer menus customizable? If not, sorry about bothering you.

**Comment by Yive** [comment]
I support this, would be really fitting to modify all menus since it'd give us the power to have a fully custom GUI compared to other modpacks.

**Comment by lumien231** [comment]
Yeeeees, but jsonifying the whole minecraft gui is quite a lot of work^^.

**Comment by iKaneki-Ken** [comment]
Ah, gotcha, If you're up for it awesome, I will patiently wait for the update that adds these options. If you're not, it's completely understandable.

**Comment by Yive** [comment]
That's true, hopefully someday the source code will be public for people to help contribute to for extra features & to speed up development processes :smile:

**Comment by Zendrex** [comment]
@lumien231 Any news on something like this?

**Comment by lumien231** [comment]
@Zendrex To be honest i probably won't add this to Custom Main Menu because that's not what Custom Main Menu is meant to do. It's for custom main menus (:P), nothing more. Also it would take too much work to actually allow any meaningful customization (Not just changing the position of "fat" elements like "resource pack list")

**Comment by Zendrex** [comment]
@lumien231 I was thinking something more along the lines of being able to change the background to a actual picture like what you can do with the main menu so it matches the style just a bit. I dont care for customizing every button, just something like the background as all.

**Comment by lumien231** [comment]
@Zendrex That's actually a good idea for [Custom Backgrounds](http://minecraft.curseforge.com/projects/custom-backgrounds?gameCategorySlug=mc-mods&projectID=225928), will add that there.

**Comment by Zendrex** [comment]
@lumien231 Would it be possible for you to add 1.7.10 support for that?
